importer/tests/test_import.py

```diff
@@ -109,3 +115,106 @@ def test_import_task(self):
+            ) as mock_static:
+                with mock.patch('importer.api.XBundle') as mock_bundle:
+                    with mock.patch('importer.api.isdir') as mock_is_dir:
+                        mock_import.return_value = True
+                        mock_is_dir.return_value = True
+                        import_course_from_path(
+                            test_path, test_repo_id, test_user_id
+                        )
+                        mock_import.assert_called_with(
+                            mock_bundle(), test_repo_id, test_user_id
+                        )
+                        mock_static.assert_called_with(
+                            os.path.join(test_path, 'static'), True
+                        )
+
+    def test_import_static_assets(self):
+        """
+        Verify walking a folder of assets and verifying they get added
+        """
+        temp_dir_path = mkdtemp()
+        self.addCleanup(rmtree, temp_dir_path)
+        basename = 'blah.txt'
+        file_contents = 'hello\n'
+        with open(os.path.join(temp_dir_path, basename), 'w') as temp:
+            temp.write(file_contents)
+        # All setup, now import
+        import_static_assets(temp_dir_path, self.course)
+        assets = StaticAsset.objects.filter(course=self.course)
+        self.assertEqual(assets.count(), 1)
+        asset = assets[0]
+        dummy = mock.MagicMock()
+        dummy.course = self.course
+        self.assertEqual(
+            asset.asset.name,
+            static_asset_basepath(dummy, basename)
+        )
+        self.addCleanup(default_storage.delete, asset.asset)
+
+    def test_import_static_recurse(self):
+        """
+        Verify walking a folder of assets and verifying they get added
+        """
+        temp_dir_path = mkdtemp()
+        self.addCleanup(rmtree, temp_dir_path)
+        basename = 'blah.txt'
+        file_contents = 'hello\n'
+        # Create folder and additional directory to verify recursion.
+        subdir_name = 'testdir'
+        os.mkdir(os.path.join(temp_dir_path, subdir_name))
+        with open(
+            os.path.join(temp_dir_path, subdir_name, basename),
+            'w'
+        ) as temp:
+            temp.write(file_contents)
+
+        # All setup, now import
+        import_static_assets(temp_dir_path, self.course)
+        assets = StaticAsset.objects.filter(course=self.course)
+        self.assertEqual(assets.count(), 1)
+        asset = assets[0]
+        dummy = mock.MagicMock()
+        dummy.course = self.course
+        self.assertEqual(
+            asset.asset.name,
+            static_asset_basepath(dummy, os.path.join(subdir_name, basename))
+        )
+        self.addCleanup(default_storage.delete, asset.asset)
+
+    def test_static_import_integration(self):
+        """
+        Do integration test to validate course import with static assets.
+        """
+        original_count = Course.objects.count()
+        import_file(
```

Comment:
That would be a test of the view right (https://github.com/mitodl/lore/blob/feature/upload_assets/ui/views.py#L134)?  That is outside of this area (and also out of scope for this PR).